pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/admin/UserAdminService.java

```diff
@@ -7,14 +7,7 @@
 import java.sql.SQLException;
 import java.sql.Statement;
 import java.sql.Types;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
Not sure this is in our checkstyle but most places I've worked prefer to not use '*'. There may be a Java best practices somewhere, but this is frowned on I believe. (And I don't think we use that pattern in DSM.)